pkg/templates/updateconfig/template_test.go

```diff
@@ -499,8 +508,14 @@ func (s *UpgradeConfigTestSuite) TestMinSurgeInteger() {
 				},
 				deploymentWithValidRollingUpdate:       {},
 				deploymentConfigWithValidRollingUpdate: {},
-				daemonSetWithStrategy:                  {},
-				replicationController:                  {},
+				daemonSetInvalidStrategy: {
+					{Message: "object has a max surge of <nil> but at least 2 is required"},
```

Comment:
@rrati Here's the case we discussed in CoreOS slack. Formerly the test was called `daemonSetWithStrategy`. I renamed it to `daemonSetInvalidStrategy`.
It used to expect no message `{}`, which, as we discussed, is wrong. Now it expects the message and the test succeeds after dependency updates from the previous commit [`7148bed` (#215)](https://github.com/stackrox/kube-linter/pull/215/commits/7148bed15124a12bdc75522b0258e66b571a5571). (I am not curious enough to trace down which update fixed DaemonSet behavior.)